### ISSUE TITLE
from_row for BinlogRow

### ISSUE THREAD

**Issue by normanrz** [blackbeam/rust_mysql_common#54]
In an old pre-release version, I was able to convert an updated row from a binlog event to a typed tuple with the `from_row` function.
```rust
let row_data: T = mysql_async::from_row(binlog_row.unwrap());
```

Now, this doesn't work anymore. The error message is:
```
mismatched types

expected struct `mysql_async::Row`, found struct `mysql_async::binlog::row::BinlogRow`rustc(E0308)
binlog.rs(49, 27): expected struct `mysql_async::Row`, found struct `mysql_async::binlog::row::BinlogRow`
```

I worked around the issue by converting the `BinlogRow` into a `Row` first: 
```rust
fn convert_binlog_row<T: FromRow + std::fmt::Debug>(
    binlog_row: Option<mysql_async::binlog::row::BinlogRow>,
) -> T {
    let row_data = binlog_row.unwrap();
    let columns = row_data.columns();
    let values_opt: Vec<Option<mysql_common::value::Value>> = row_data
        .unwrap()
        .into_iter()
        .map(|binlog_value| match binlog_value {
            mysql_common::binlog::value::BinlogValue::Value(value) => Some(value),
            _ => None,
        })
        .collect();
    let row_data: T = mysql_async::from_row(mysql_common::row::new_row_raw(values_opt, columns));
    row_data
}
```

I was wondering if there is a better way to do this?

**Comment by blackbeam** [comment]
Hi.
Yeah, there is no direct correspondence between `Value` and `BinlogValue` (because of JsonDiff, I think), so the lossy conversion like in your `convert_binlog_row` function seem to be the only way to do this. I believe I need to add a similar function into the library.

**Comment by blackbeam** [comment]
Partially addressed in [v0.28.0](https://github.com/blackbeam/rust_mysql_common/releases/tag/v0.28.2). Still not sure whether `BinlogRow` and `BinlogValue` should have it's own set of conversion functions 🤔

**Comment by normanrz** [comment]
Thanks! I am now using it in my code.
